README.md

```diff
@@ -559,6 +560,249 @@ The model can also be downloaded and served locally.
+$ ilab model evaluate --benchmark mmlu_branch --model $ILAB_MODELS_DIR/instructlab/granite-7b-lab --base-model $ILAB_MODELS_DIR/instructlab/granite-7b-lab --tasks-dir $ILAB_TASKS_DIR
+...
+# KNOWLEDGE EVALUATION REPORT
+
+## BASE MODEL
+/home/example-user/.local/share/instructlab/models/instructlab/granite-7b-lab
+
+## MODEL
+/home/example-user/.local/share/instructlab/models/instructlab/granite-7b-lab
+
+### AVERAGE:
++0.0 (across 1)
+
+### NO CHANGE:
+1. tonsils
+```
+
+Below is an example of running MMLUBranch with Hugging Face models:
+
+```bash
+$ export ILAB_TAXONOMY_DIR=/home/example/.local/share/instructlab/taxonomy
+$ export ILAB_TASKS_DIR=/home/example-user/.local/share/instructlab/taxonomy
+$ ilab model evaluate --benchmark mmlu_branch --model instructlab/granite-7b-lab --base-model instructlab/granite-7b-lab --tasks-dir $ILAB_TASKS_DIR
+...
+# KNOWLEDGE EVALUATION REPORT
+
+## BASE MODEL
+instructlab/granite-7b-lab
+
+## MODEL
+instructlab/granite-7b-lab
+
+### AVERAGE:
++0.0 (across 1)
+
+### NO CHANGE:
+1. tonsils
+```
+
+> [!TIP]
+> You can mix and match running local models and remote models on Hugging Face, so long as the nessessary safetensors are present.
+
+#### Running MTBench
+
+Below is an example of running MTBench with local safetensor models:
+
+```bash
+$ export ILAB_MODELS_DIR=/home/example-user/.local/share/instructlab/models
+$ ilab model evaluate --benchmark mt_bench --model $ILAB_MODELS_DIR/instructlab/granite-7b-lab --judge-model $ILAB_MODELS_DIR/instructlab/granite-7b-lab
+...
+# SKILL EVALUATION REPORT
+
+## MODEL
+/home/example-user/.local/share/instructlab/models/instructlab/granite-7b-lab
+
+### AVERAGE:
+8.07 (across 91)
+
+### TURN ONE:
+8.64
+
+### TURN TWO:
+7.19
+
+### ERROR RATE:
+0.43
+```
+
+Below is an example of running MTBench with local GGUF models:
+
+```bash
+$ export ILAB_MODELS_DIR=/home/example-user/.local/share/instructlab/models
+$ ilab model evaluate --benchmark mt_bench --model $ILAB_MODELS_DIR/granite-7b-lab-Q4_K_M.gguf --judge-model $ILAB_MODELS_DIR/granite-7b-lab-Q4_K_M.gguf
+...
+# SKILL EVALUATION REPORT
+
+## MODEL
+/home/example/.local/share/instructlab/models/granite-7b-lab-Q4_K_M.gguf
+
+### AVERAGE:
+5.0 (across 1)
+
+### TURN ONE:
+5.0
+
+### TURN TWO:
+N/A
+
+### ERROR RATE:
+0.99
+```
+
+> [!NOTE]
+> Currently, MTBench must be used with local models. Using models directly from Hugging Face without downloading them is unsupported.
+
+#### Running MTBenchBranch
+
+Below is an example of running MTBenchBranch with local safetensor models:
+
+```bash
+$ export ILAB_MODELS_DIR=/home/example-user/.local/share/instructlab/models
+$ export ILAB_TAXONOMY_DIR=/home/example-user/.local/share/instructlab/taxonomy
+$ ilab model evaluate --benchmark mt_bench_branch \
+   --model $ILAB_MODELS_DIR/instructlab/granite-7b-lab \
+   --judge-model $ILAB_MODELS_DIR/instructlab/granite-7b-lab \
+   --base-model $ILAB_MODELS_DIR/instructlab/granite-7b-lab \
+   --taxonomy-path $ILAB_TAXONOMY_DIR \
```

Comment:
The command above did not pass `--taxonomy_path` or `--branch`. What's special about this command that these **are** required?

Reply:
MTBenchBranch is used to compare the evaluation score diff between two different branches of a taxonomy tree, whereas MTBench just evaluates a given model with a score